src/components/VitalsSideBar.vue

```diff
@@ -10,60 +10,69 @@ import VitalStat from './VitalStat.vue'
  * @param bloodPotency {JSON} dependency, @TODO
  */
 export default {
-  props: {'vitals':JSON, 'stamina': Number, 'composure':Number,'resolve':Number,'bloodPotency':Number,},
-  emits:['hover'],
-  components:{
-    "vital-stat":VitalStat,
+  props: {
+    vitals: JSON,
+    stamina: Number,
+    composure: Number,
+    resolve: Number,
+    bloodPotency: Number,
+  },
+  emits: ['hover'],
+  components: {
+    'vital-stat': VitalStat,
   },
   computed: {
     /** Computes values of attributes to send to vital stat component */
-    values(){
-      var valueArray = []
-      
-      valueArray["Health"] = this.dependecySum([this.stamina])
-      valueArray["Willpower"] = this.dependecySum([this.resolve + this.composure])
-      valueArray["Humanity"] = this.dependecySum([])
-      valueArray["Hunger"] = this.dependecySum([])
-      valueArray["Blood Potency"] = this.dependecySum([])
-      
-      return valueArray      
-    }
+    values() {
+      var valueArray = [];
+
+      valueArray['Health'] = this.dependecySum([this.stamina]);
+      valueArray['Willpower'] = this.dependecySum([
+        this.resolve + this.composure,
```

Comment:
ugh.. tam mala byt ciarka, a neviem preco tam je plus

Reply:
aj tak to neberie dependencies z data.json a aj s ciarkou je to len funkcia navyse, ktora to len komplikuje